src/modules/emote_menu/components/Button.jsx

```diff
@@ -0,0 +1,36 @@
+import React, {useEffect, useRef, useState} from 'react';
+import {faSmile} from '@fortawesome/free-solid-svg-icons/faSmile';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
+import Whisper from 'rsuite/lib/Whisper/index.js';
+import Popover from 'rsuite/lib/Popover/index.js';
+import EmoteMenu from './Menu.jsx';
+import styles from '../styles/button.module.css';
+import emoteStore from '../stores/index.js';
+
+export default function Button({appendText}) {
+  const triggerRef = useRef(null);
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    emoteStore.once('updated', () => setLoading(false));
+  }, []);
+
+  return (
+    <Whisper
+      trigger="click"
+      placement="topEnd"
+      speaker={
+        <Popover
+          className={styles.popover}
+          style={{width: window.location.pathname.endsWith('/chat') ? 320 : 380}}
```

Comment:
can we avoid hardcoding this in js? is it possible to use css to handle the window size < 380?